.github/workflows/lango-stale-reviews.yml

```diff
@@ -0,0 +1,25 @@
+name: "lango-stale-reviews"
+on:
+  schedule:
+    - cron:  '* */1 * * *'    # At every hour.
+
+jobs:
+  update-last-activity-dates:
+    env:
+      LANGO_REVIEW_BOARD_TOKEN: ${{ github.token }}
+      ORGANIZATION: Tarantool
+      PROJECT_ID: 83
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python3
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+          cache: 'pip'
```

Comment:
Quotes can be omitted.

Reply:
BTW, quotes around `3.9` are desirable, because it is easy to forget about them for, say, `3.10` and get `3.1`.